with-snowflake/README.md

```diff
@@ -0,0 +1,315 @@
+# StepZen Example: `with-snowflake`
+
+## Introduction
+
+This project builds a GraphQL API connected to your Snowflake warehouse using StepZen.
+We will use the `@dbquery` directive to generate our GraphQL schema.
+
+## Getting Started
+
+You'll need to create a [StepZen account](https://stepzen.com/request-invite) first. Once you've got that set up, [git clone](https://www.atlassian.com/git/tutorials/setting-up-a-repository/git-clone) this repository onto your machine and open the working directory:
+
+```bash
+git clone https://github.com/stepzen-dev/examples.git
+cd examples/with-snowflake
+```
+
+Before you can run this example, you need to have create a Snowflake warehouse following
```

Comment:
"to have create" ->. "to create" or "to have created"

Reply:
OR, since you say this below, why not say something different here.

To run this example you will need a StepZen account and a Snowflake warehouse, but first, [git clone](https://www.atlassian.com/git/tutorials/setting-up-a-repository/git-clone) this repository onto your machine and open the working directory:

```bash
git clone https://github.com/stepzen-dev/examples.git
cd examples/with-snowflake
```

Now create a [StepZen account](https://stepzen.com/request-invite). Once you've got that set up, follow the next section to create a Snowflake warehouse.